src/components/forms/pool_actions/InvestForm/components/InvestPreviewModal/components/InvestActionsV2.vue

```diff
@@ -59,23 +57,11 @@ const {
 
 const approvalActions = ref(joinPoolApprovalActions.value);
 
-const tokensToApprove = computed(() =>
-  amountsIn.value.map(amountIn => amountIn.address)
-);
-const amountsToApprove = computed(() =>
-  amountsIn.value.map(amountIn => amountIn.value)
-);
-const { tokenApprovalActions } = useTokenApprovalActions(
-  tokensToApprove.value,
-  amountsToApprove
-);
-
 /**
  * COMPUTED
  */
 const actions = computed((): TransactionActionInfo[] => [
   ...approvalActions.value,
-  ...tokenApprovalActions,
```

Comment:
Why would you remove this from here? approvalActions and tokenApprovalActions are different things. Where are the token approvals loaded now?

Reply:
Token approvals and relayer approval both are now part of the `approvalActions` that comes from  [join-pool-provider]( https://github.com/balancer-labs/frontend-v2/pull/2800/files#diff-c5466dc8169d416c1cd54c2bc0e3e8d6e64dae65d71bdc35675bd48b3c3dca87R211).

Provider needs to know about token approvals, so it knows when to use right simulation type.